src/equations/equations.jl

```diff
@@ -210,12 +323,27 @@ Default analysis integrals used by the [`AnalysisCallback`](@ref).
 """
 default_analysis_integrals(::AbstractEquations) = Symbol[]
 
+abstract type AbstractBathymetry end
+struct BathymetryFlat <: AbstractBathymetry end
+"""
+    bathymetry_flat = DispersiveShallowWater.BathymetryFlat()
+
+A singleton struct indicating a flat bathymetry ``b = 0``.
```

Comment:
```suggestion
A singleton struct indicating a flat bathymetry.
```
In our current approach it doesn't need to be 0, does it?